.github/workflows/main.yml

```diff
@@ -19,9 +19,9 @@ jobs:
           - name: 'Python 3.6.10'
             os: ubuntu-18.04
             python-version: '3.6.10'
-          - name: 'Python 3.7.7'
+          - name: 'Python 3.7.9'
             os: ubuntu-18.04
-            python-version: '3.7.7'
+            python-version: '3.7.9'
```

Comment:
@Edilmo I have changed the python version because the test build was failing.

Reply:
Sounds good to me. I had problems with this as well